run.sh

```diff
@@ -40,6 +40,10 @@ get_safe_name(){
       name=$(echo "$name" | sed 's/_/-/g')
       ;;
 
+    unsafe)
+      # Don't replace any character, even if invalid for RFC 1123
```

Comment:
where's the implementation? :sweat_smile: 

If you don't do it, I'll convert this into an issue (feat. request)